src/ploneintranet/workspace/tests/workspace.robot

```diff
@@ -12,61 +12,69 @@ Test Teardown  Close all browsers
 
 *** Test Cases ***
 
-Site Administrator can add example user as member of workspace
-    Log in as site owner
-    Add workspace  Example Workspace
-    Maneuver to  Example Workspace
-    Click Link  jquery=a:contains('View full Roster')
-    Input text  edit-roster-user-search  Example User
-    Click button  Search users
+# I outcomment these tests as they were written for barceloneta.
```

Comment:
Commented code should not be committed like this. If you need a reference, use the git history of this file.